AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -318,6 +318,7 @@ investkorea.org##.cookie-area
 rapfame.app###header > div[class^="MuiCollapse-container-"][style^="min-height:"]
 eltiempo.com##.c-popup--politicas
 traghettilines.it###div_CookiePolicy
+shop.leet.hu#@#.cookie-alert
```

Comment:
Unblocking cookies should be moved to `cookies_allowlist` so that the window is not blocked with AdGuard content blocker and AdGuard ios free

You also need to block cookies with `#$#` - css rules, for more powerful AdGuards